packages/beacon-node/package.json

```diff
@@ -97,6 +97,7 @@
     "@chainsafe/as-chacha20poly1305": "^0.1.0",
     "@chainsafe/as-sha256": "^0.3.1",
     "@chainsafe/bls": "7.1.1",
+    "@chainsafe/blst": "^0.2.9",
```

Comment:
This was just missed I guess, gets rid of `warning "workspace-aggregator-7fe22546-d232-4ae9-9884-df7a13280865 > @lodestar/beacon-node > @chainsafe/bls@7.1.1" has unmet peer dependency "@chainsafe/blst@^0.2.4".`